src/content/en/2020/markup.md

```diff
@@ -543,58 +607,64 @@ When we built our tests we didn’t check for the presence of images, but only l
 | … | | |
 | SVG | 5,328 | 0.08% |
 
-<figcaption>{{ figure_link(caption="Common favicon approaches.", sheets_gid="1930085905", sql_file="pages_almanac_by_device_and_favicon_image_type.sql") }}</figcaption>
+<figcaption>{{ figure_link(caption="Common favicon formats.", sheets_gid="1930085905", sql_file="pages_almanac_by_device_and_favicon_image_type.sql") }}</figcaption>
 </figure>
 
-There may be a couple of surprises in here:
+There are a couple of surprises in here:
 
-* Support for other formats is there but ICO is still the go-to format for favicons on the Web.
+* Support for other formats is there but ICO is still the go-to format for favicons on the web.
 * JPG is a relatively popular favicon format even though it may not yield the best results (or a comparatively large file size) for many favicon sizes.
-* WebP is—twice as popular as SVG! (This might change, however, with [SVG favicon support improving](https://caniuse.com/link-icon-svg).)
+* WebP is twice as popular as SVG! This might change, however, with [SVG favicon support](https://caniuse.com/link-icon-svg) improving.
 
 ### Button and input types
 
-There has been a lot of discussion on buttons lately and how often they are misused. We looked into this to present findings on some of the native HTML buttons:
+{# TODO(authors): Can you cite any examples of this discussion? And what do the findings say about button misuse? #}
```

Comment:
I don’t know about discussions but people write and talk a lot about it.

Vadim Makeev 2019: https://www.smashingmagazine.com/2019/02/buttons-interfaces/
HTMHell 2020: https://www.htmhell.dev/20-close-buttons/
Marcy Sutton 2017: https://www.youtube.com/watch?v=8XjwDq9zG4I

Reply:
Thanks for adding these helpful resources @matuzo and I'd like to add these to the conversation as well:

- Links, Buttons, Submits, and Divs, Oh Hell
  https://adrianroselli.com/2016/01/links-buttons-submits-and-divs-oh-hell.html

- Links are not buttons. Neither are DIVs and SPANs
  https://karlgroves.com/2013/05/14/links-are-not-buttons-neither-are-divs-and-spans

Reply:
I'd suggest picking the most representative example and link to it from the word "discussion". I can apply that change if you let me know which URL you'd prefer.

Reply:
I'd suggest linking Adrian Roselli's write-up as it contains lots of updates and links to other similar resources too.